### ISSUE TITLE
ghcr image of v0.1.8 does not start up on OpenShift 

### ISSUE THREAD

**Issue by Hannessi** [interlynk-io/sbomqs#319]
A container of the new version v0.1.8 does not start up on OpenShift. 

The container is used as part of a GitLab pipeline (hence within a runner), however on container startup the application fails with error message, 

```Error: container create failed: time="2024-08-22T11:28:22Z" level=error msg="runc create failed: unable to start container process: exec: \"sh\": executable file not found in $PATH"```

Stage details: 

```score sbom:
  stage: sbom
  needs:
    - upload sbom
  image: ${GHCR_PROXY}interlynk-io/sbomqs:v1.8.0
  variables:
    GIT_STRATEGY: none
    QUALITY_SCORE_REPORT_FILENAME: "sbom-quality-score-report.txt"
  script:
    - /app/sbomqs dtrackScore --url "https://${DEPENDENCY_TRACK_URL}/" --api-key "${DEPENDENCY_TRACK_API_KEY}" ${DEPENDENCY_TRACK_PROJECT_ID} --tag-project-with-score > "$QUALITY_SCORE_REPORT_FILENAME"
    - cat "$QUALITY_SCORE_REPORT_FILENAME"
  artifacts:
    paths:
      - "$QUALITY_SCORE_REPORT_FILENAME"
  rules:
    - if: $PIPELINE == "Deploy" && $CUSTOM_TAG == "" && $CI_COMMIT_REF_NAME == "main"
```

Openshift version 4.14
Kubernetes version 1.27.15

**Comment by riteshnoronha** [comment]
@Hannessi this has been fixed in v0.1.9 please give it a try. I used this to test on gitlab

```
  34   │ sbomqs:
  35   │   stage: test
  36   │   image:
  37   │     name: ghcr.io/interlynk-io/sbomqs:v0.1.9
  38   │     entrypoint: [""]
  39   │   needs: ["sbom"]
  40   │   script:
  41   │     - /app/sbomqs score syft.cdx.json
```

**Comment by Hannessi** [comment]
Working perfectly! Thanks so much!
